staging/src/k8s.io/apiserver/pkg/registry/generic/registry/storage_factory.go

```diff
@@ -49,7 +50,7 @@ func StorageWithCacher(capacity int) generic.StorageDecorator {
 
 		// TODO: we would change this later to make storage always have cacher and hide low level KV layer inside.
 		// Currently it has two layers of same storage interface -- cacher and low level kv.
-		cacherConfig := storage.CacherConfig{
```

Comment:
finally the cacher is easily to find 👍

Reply:
yes, when I read the cacher sourcecode and found a separated folder is more clear. Files under storage should be common.